romana-install/roles/romana/install-agent/templates/config/agent_proxy_add_policy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+body=`echo $1 | cut -d'=' -f2`
+echo Posting $body to http://{{ romana_master_ip }}:9630/
```

Comment:
OK. So these can be plain files instead of templates and the reference to romana_master_ip can be changed to localhost.